src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1610,6 +1619,9 @@ void goto_instrument_parse_optionst::help()
     " --render-cluster-function    clusterises the dot by functions\n"
     "\n"
     "Slicing:\n"
+    " --fp-reachability-slice <f>  remove instructions that cannot appear on a\n" // NOLINT(*)
```

Comment:
`--fp-reachability-slice f`
`The list of functions has to be given as a comma-separated list f.`

Reply:
Shouldn't this use `HELP_REACHABILITY_SLICER` from below?

Reply:
It seems that this help macro is for CBMC options, e.g. `--reachability-slice-fb` is not a valid option for `goto-instrument`.